lib/senkyoshi/models/wikipage.rb

```diff
@@ -30,5 +38,31 @@ def canvas_conversion(course, resources)
 
       course
     end
+
+    def _extendeddata(extendeddata)
+      Nokogiri::XML(extendeddata).
+        search("LessonPlanComponent").
+        map do |node|
+          visible = true?(node.search("vislableToStudents").attr("value").to_s)
```

Comment:
Another approach to this would be to use an xpath search. I think it looks a bit, but I suppose that is pretty subjective. `node.xpath("./vislableToStudents/@value").text`